src/logger.ts

```diff
@@ -132,7 +133,7 @@ function initLogging (): void {
       upperCaseLevel(),
       // format.padLevels(),
       format.timestamp({ format: 'DD/MM hh:mm:ss' }),
-      format.colorize(),
+      !supportsColor() || process.env.LOG_NO_COLORS === 'true' ? format(i => i)() : format.colorize(),
```

Comment:
The chunk below has a hardcoded `colors.grey`, which I think is not going to be affected by this changes.

Reply:
And from my probably limited understanding of how color is being handled by the library, https://github.com/winstonjs/logform/blob/master/colorize.js#L9 enables colors under any circumstance, offering no way of make `colors.grey` a no-op.

Reply:
Hmm thanks for pointing this out, I missed that one 😅

In the end I choose instead to put `uncolorize()` there as we have some colored input coming in directly from our code, so this might not be as optimal, but will work.